inbm/dispatcher-agent/dispatcher/aota/aota_command.py

```diff
@@ -452,7 +454,7 @@ def _download(self) -> None:
             f'Package: {self._uri} Fetch Result: {get_result}')
 
         if get_result.status != CODE_OK:
-            raise AotaError('Unable to download docker-compose container.')
+            raise AotaError(f"Unable to download docker-compose container. Status = {get_result.status}, message = {get_result.message}.")
```

Comment:
**<h2>$\color{#FFB6C1}Enhancement: CodeRefactoring$</h2>**
**Suggestion:** Refactor the repeated error-raising logic into a separate method to reduce code duplication and improve maintainability.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        self._handle_fetch_error(get_result)
```